modules/claims_api/app/controllers/claims_api/v2/veteran_identifier_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'evss/error_middleware'
+
+module ClaimsApi
+  module V2
+    class VeteranIdentifierController < ClaimsApi::V2::Base
+      skip_before_action :authenticate, only: %i[create]
+      ICN_FOR_TEST_USER = '1012667145V762142'
+      FORM_NUMBER = 'VETERAN_IDENTIFIER'
```

Comment:
can we rename this to be something more generic? this isnt a form...

Reply:
`FORM_NUMBER` is what `v0` and `v1` use for identifying what schema file name to use. ([ref](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/controllers/claims_api/v1/forms/base.rb#L20))
We can change this `v2` implementation to a diff name if we need to, but I err on the side of staying consistent.

Reply:
i get that, but v0 and v1 only had posts within the context of "forms"
id argue this should be changed for v2 because its more than just forms in this version

Reply:
looking at what all this extends, im now wondering if i was just misunderstand form in this context?
the name "form" is at the core of vets-api, so i wonder if this is more like "form == request" sort of?
either way, this is probably not worth trying to rename since it is in a core class.